generated/nidaqmx/_task_modules/channels/ao_channel.py

```diff
@@ -1969,8 +1971,8 @@ def ao_resolution_units(self):
     @property
     def ao_term_cfg(self):
         """
-        :class:`nidaqmx.constants.TerminalConfiguration`: Specifies the
-            terminal configuration of the channel.
+        :class:`nidaqmx.constants.OutputTermCfg`: Specifies the terminal
```

Comment:
Its still `TerminalConfiguration` in #215

Reply:
Updated.